code/modules/cargo/packs.dm

```diff
@@ -2478,6 +2478,13 @@
 	contains = list(/mob/living/simple_animal/hostile/retaliate/goat)
 	crate_name = "goat crate"
 
+/datum/supply_pack/critter/mothroach
+	name = "Mothroach Crate"
+	desc = "Put the mothroach on your head and find out what true cuteness looks like."
+	cost = 1500
```

Comment:
Feels a bit cheap. Maybe 5000 like the other pets?

Reply:
Fair enough!